lesson3Test/lesson3.js

```diff
@@ -0,0 +1,12 @@
+let b="######"
+
+
+for(let i=0; i<=b.length; i++){
```

Comment:
ეს for ციკლი არ მომწონს, შეცვალე და გამოიყენე While ციკლი იგივე ოპერაციის შესასრულებლად